File: 백준/Silver/1475. 방 번호/방 번호.java
import java.io.*;
import java.util.*;

public class Main{
    public static void main(String[] args) throws IOException{
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        String[] nArr = br.readLine().split("");
        int[] arr = new int[10];
        
        for(int i=0; i<nArr.length; i++){
            if(nArr[i].equals("6")){
                arr[9]++;
            }else{
                arr[Integer.parseInt(nArr[i])]++;
            }
        }
        if(arr[9]%2!=0){ //99999 -> 3세트 필요
            arr[9] = arr[9]/2+1;
        }else{
            arr[9]/=2;
        }
        int answer = 0;
        for(int i=0; i<arr.length; i++){
            answer = Math.max(answer, arr[i]);
        }
        br.close();
        System.out.println(answer);
    }
 }

File: 백준/Silver/1475. 방 번호/방 번호.py
word = input()
answer = [0] * 10
for i in range(len(word)):
    num = int(word[i])
    if num == 6 or num == 9:
        if answer[6] <= answer[9]:
            answer[6] += 1
        else:
            answer[9] += 1
    else:
        answer[num] += 1
 
print(max(answer))